model/src/main/java/org/mskcc/cmo/metadb/model/web/PublishedMetadbSample.java

```diff
@@ -0,0 +1,506 @@
+
+    public String getPreservation() {
+        return preservation;
+    }
+
+    public void setPreservation(String preservation) {
+        this.preservation = preservation;
+    }
+
+    public String getTumorType() {
+        return tumorType;
+    }
+
+    public void setTumorType(String tumorType) {
+        this.tumorType = tumorType;
+    }
+
+    public String getParentTumorType() {
+        return parentTumorType;
+    }
+
+    public void setParentTumorType(String parentTumorType) {
+        this.parentTumorType = parentTumorType;
+    }
+
+    public String getSpecimenType() {
+        return specimenType;
+    }
+
+    public void setSpecimenType(String specimenType) {
+        this.specimenType = specimenType;
+    }
+
+    public String getSampleOrigin() {
+        return sampleOrigin;
+    }
+
+    public void setSampleOrigin(String sampleOrigin) {
+        this.sampleOrigin = sampleOrigin;
+    }
+
+    public String getTissueSource() {
+        return tissueSource;
+    }
+
+    public void setTissueSource(String tissueSource) {
+        this.tissueSource = tissueSource;
+    }
+
+    public String getTissueLocation() {
+        return tissueLocation;
+    }
+
+    public void setTissueLocation(String tissueLocation) {
+        this.tissueLocation = tissueLocation;
+    }
+
+    public String getRecipe() {
+        return recipe;
+    }
+
+    public void setRecipe(String recipe) {
+        this.recipe = recipe;
+    }
+
+    public String getBaitSet() {
+        return baitSet;
+    }
+
+    public void setBaitSet(String baitSet) {
+        this.baitSet = baitSet;
+    }
+
+    public String getFastqPath() {
+        return fastqPath;
+    }
+
+    public void setFastqPath(String fastqPath) {
+        this.fastqPath = fastqPath;
+    }
+
+    public String getPrincipalInvestigator() {
+        return principalInvestigator;
+    }
+
+    public void setPrincipalInvestigator(String principalInvestigator) {
+        this.principalInvestigator = principalInvestigator;
+    }
+
+    public String getAncestorSample() {
+        return ancestorSample;
+    }
+
+    public void setAncestorSample(String ancestorSample) {
+        this.ancestorSample = ancestorSample;
+    }
+
+    public String getSampleStatus() {
+        return sampleStatus;
+    }
+
+    public void setSampleStatus(String sampleStatus) {
+        this.sampleStatus = sampleStatus;
+    }
+
+    public String getRequestId() {
+        return requestId;
+    }
+
+    public void setRequestId(String requestId) {
+        this.requestId = requestId;
+    }
+
+    public Map<String, String> getCmoSampleIdFields() {
+        return cmoSampleIdFields;
+    }
+
+    public void setCmoSampleIdFields(Map<String, String> cmoSampleIdFields) {
+        this.cmoSampleIdFields = cmoSampleIdFields;
+    }
+
+    public void setSampleAliases(List<SampleAlias> sampleAliases) {
+        this.sampleAliases = sampleAliases;
+    }
+
+    public List<SampleAlias> getSampleAliases() {
```

Comment:
`getSampelAliases` should have same logic as other getters that return lists like [here](https://github.com/mskcc/cmo-metadb/pull/468/files#diff-34b0ee699f8dfd66575470b982c99631815ba2d9bd342263d973e14e82e0e8f6R277)